data/pools/transactionPool.go

```diff
@@ -79,7 +79,7 @@ func MakeTransactionPool(ledger *ledger.Ledger, cfg config.Local) *TransactionPo
 		expiredTxCount:  make(map[basics.Round]int),
 		ledger:          ledger,
 		statusCache:     makeStatusCache(cfg.TxPoolSize),
-		logStats:        cfg.EnableAssembleStats,
+		logStats:        cfg.EnableProcessBlockStats,
```

Comment:
nit: a better name for this variable would be enableLogStats, but it's beyond the scope of your change.